etc/cmake/options.cmake

```diff
@@ -42,7 +42,9 @@ endif()
 
 option(OTBR_BORDER_ROUTING "Enable Border Routing Manager" OFF)
 
-option(OTBR_BORDER_ROUTING_NAT64 "Enable NAT64 support in Border Routing Manager" OFF)
+option(OTBR_BORDER_ROUTING_NAT64 "Enable NAT64 border routing support" OFF)
```

Comment:
nit: add "deprecated" or "deprecating" to the description to avoid confusion before it's fully removed